how-to-docs/certificates.md

```diff
@@ -179,3 +179,7 @@ spec:
           serviceName: myservicename
           servicePort: 443
 ```
+
+**LetsEncrypt Limits**
+
+Note Letscrypt while a free service does come with a number fo service limit detailed [here](https://letsencrypt.org/docs/rate-limits/). Probably one of the most crucial for projects is the max certificate requests per week; currently standing out 20. In addition, there is a max 5 failures for per hostname with a freeze of 1 hour, so if you accidently mess up configuration you might hit this.
```

Comment:
commas around the subordinate conjunction pls
typo: of
typo: limits
typo: standing at